bigquery_etl/docs/generate_docs.py

```diff
@@ -150,7 +152,7 @@ def main():
                                 dataset_doc_file.write(f"{sourced}\n\n")
                         else:
                             # dataset-level doc; create a new doc file
-                            dest = out_dir / path / f"{name}.md"
+                            dest = Path(out_dir) / "mozfun" / f"{name}.md"
```

Comment:
Now that we're using `tmp/generated-sql` dir, we need to be more specific with the destination folder name